src/test/java/com/internship/frejaeidjmeterplugin/jmeter/frejaRequests/SignServiceTest.java

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
Remove this comments. You can google how to configure Netbeans not to generate you this every time you create new class.